pkg/virt-controller/services/template.go

```diff
@@ -173,6 +173,19 @@ func (t *templateService) RenderLaunchManifest(vm *v1.VirtualMachine) (*k8sv1.Po
 		}
 	}
 
+	if len(requestedHookSidecarList) != 0 {
+		volumes = append(volumes, k8sv1.Volume{
+			Name: "hooks",
```

Comment:
I wonder if this name can be more descriptive.

Reply:
Done